index.js

```diff
@@ -0,0 +1,116 @@
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    liveLink: 'https://aliatayee.github.io/Portfolio-mobile-version-skeleton-/',
+    github: 'https://github.com/aliatayee/Portfolio-mobile-version-skeleton-',
+    techstack: ['Ruby on rails', 'CSS', 'JavaScript', 'HTML'],
+  },
+  {
+    id: 4,
+    title: 'Multi-Post Stories Gain+Glory',
+    img: 'assets/images/placeholder-image.png',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    liveLink: 'https://aliatayee.github.io/Portfolio-mobile-version-skeleton-/',
+    github: 'https://github.com/aliatayee/Portfolio-mobile-version-skeleton-',
+    techstack: ['Ruby on rails', 'CSS', 'JavaScript', 'HTML'],
+  },
+  {
+    id: 4,
+    title: 'Multi-Post Stories Gain+Glory',
+    img: 'assets/images/placeholder-image.png',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    liveLink: 'https://aliatayee.github.io/Portfolio-mobile-version-skeleton-/',
+    github: 'https://github.com/aliatayee/Portfolio-mobile-version-skeleton-',
+    techstack: ['Ruby on rails', 'CSS', 'JavaScript', 'HTML'],
+  },
+
+];
+
+window.onload = () => {
+  projects.forEach((project, index) => {
+    document.getElementById('project_container').innerHTML += `<div class="img-patholder">
+                    <div class="img-box"><img src="${project.img}" alt="project 1"></div>
+                    <h3 class="title-project">${project.title}</h3>
+                    <ul class="buttons">
+                        <li>ruby on rails</li>
+                        <li>Css</li>
+                        <li>javascript</li>
+                        <li>html</li>
+                    </ul>
```

Comment:
- [ ] Kindly use data from the project array to populate the list of technologies